src/bp/ui-studio/src/web/views/OneFlow/diagram/index.tsx

```diff
@@ -353,7 +354,15 @@ class Diagram extends Component<Props> {
     },
     executeNode: (point: Point, moreProps) =>
       this.props.createFlowNode({ ...point, type: 'execute', next: [defaultTransition], ...moreProps }),
-    routerNode: (point: Point) => this.props.createFlowNode({ ...point, type: 'router' }),
+    routerNode: (point: Point) =>
+      this.props.createFlowNode({
+        ...point,
+        type: 'router',
+        next: [
+          { condition: 'false', node: '' },
+          { condition: 'true', node: '' }
```

Comment:
Can we start with one empty condition, and add others on the go ? If you check the Trigger node, we already have the "add condition" event.

Feels weird selecting a number variable and seeing "false, otherwise"